File: 2013 ACM-ICPC North American Qualifier/oddities.java
import java.util.Scanner;

public class oddities {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        
        for (int i = 0; i < n; i++) {
            int next = scanner.nextInt();
            
            if ((next & 1) == 1) {
                System.out.println(next + " is odd");
            } else {
                System.out.println(next + " is even");
            }
        }
        
        scanner.close();
    }
}

File: 2013 ACM-ICPC North American Qualifier/oddities.pas
program oddities;
var
  n, i, next: integer;
begin
  readln(n);
  
  for i := 1 to n do
  begin
    readln(next);
    
    if (next and 1) = 1 then
      writeln(next, ' is odd')
    else
      writeln(next, ' is even');
  end;
end.